### ISSUE TITLE
keep position？

### ISSUE THREAD

**Issue by jyy1082** [ngokevin/kframe#115]
I would like to keep the position and direction of all parts from glTF file. Does this component support this? Thanks

**Comment by ngokevin** [comment]
Yeah, it could be done. I'd probably have it check if there is a position or rotation defined. If not, then use the position/rotation from the file. I won't be able to get around to it myself though.

**Comment by jyy1082** [comment]
Actually，our requirement is to import every part from glTF file as an individual entity in aFrame and keep all parts' positions&rotations. Could you tell me how can I implement it using your component? Thanks.

**Comment by ngokevin** [comment]
Would be easy enough to fork mine and make your own component. You'll need to modify it.
